examples/local/README.md

```diff
@@ -0,0 +1,117 @@
+# Running cert-operator Locally
+
+**Note:** This should only be used for testing and development. See the
+[/kubernetes/][kubernetes-dir] directory and [Secrets][secrets-doc] for
+a production ready configuration.
+
+[kubernetes-dir]: https://github.com/giantswarm/cert-operator/tree/master/kubernetes
+[secrests-doc]: https://github.com/giantswarm/cert-operator#secrets
+
+This guide explains how to get running cert-operator locally. For example on
+minikube. Certificates created here are meant to be used by [aws-operator].
+
+All commands are assumed to be run from `examples/local` directory.
+
+[aws-operator]: https://github.com/giantswarm/aws-operator
+
+
+## Preparing Templates
+
+All yaml files in this directory are templates. Before proceeding this guide
+all placeholders must be replaced with sensible values.
+
+- *CLUSTER_NAME* - Cluster name to be created by [aws-operator].
+- *COMMON_DOMAIN* - Cluster name to be created by [aws-operator].
```

Comment:
@pawel the ELB is in the gigantic.io domain because its serving customer data. We use a separate domain for security.

We don't provide a cert for the ingress so it shouldn't be needed in the examples.